tutorandroid/plugin.py

```diff
@@ -2,11 +2,10 @@
 import os
 import pkg_resources
 
-from tutor import hooks as tutor_hooks
+from tutor import hooks as tutor_hooks, hooks, exceptions
```

Comment:
Use `tutor_hooks` and do not import `hooks`.

Reply:
Done!